keps/sig-auth/541-external-credential-providers/README.md

```diff
@@ -493,6 +493,114 @@ credentials).
 PEM format. The certificate must be valid at the time of execution. These
 credentials are used for mTLS handshakes.
 
+### Metrics
+
+As discussed [below](#rollout-upgrade-and-rollback-planning), there are 4
+primary metrics used by this feature set.
+
+```golang
+var (
+  execPluginCertTTL = k8smetrics.NewGaugeFunc(
```

Comment:
Should we put a `"command"` label on these exec plugin metrics to partition them by the underlying exec binary/command? How will a user of a single client-go process with multiple exec plugins differentiate between metrics about one exec plugin from another?

Reply:
unless we include all the inputs (envvars, args, etc), which we wouldn't do for security and cardinality reasons, it seems likely we wouldn't be able to reliably distinguish different invocations.